core/src/main/java/org/elasticsearch/index/query/MultiMatchQueryBuilder.java

```diff
@@ -230,6 +254,11 @@ public MultiMatchQueryBuilder prefixLength(int prefixLength) {
         return this;
     }
 
+    public MultiMatchQueryBuilder prefixLength(Integer prefixLength) {
+        this.prefixLength = prefixLength;
```

Comment:
Same here, we can change to primitive `int` and use default values for this and some more of the fuzzy options. This will also simplify doXContent, can also be used in parser then.